plutus-chain-index-core/src/Plutus/ChainIndex/Handlers.hs

```diff
@@ -82,7 +82,6 @@ handleQuery = \case
     MintingPolicyFromHash hash  -> getScriptFromHash hash
     RedeemerFromHash hash       -> getRedeemerFromHash hash
     StakeValidatorFromHash hash -> getScriptFromHash hash
-    TxFromTxId txId             -> getTxFromTxId txId
     TxOutFromRef tor            -> getTxOutFromRef tor
```

Comment:
I would change this to `UnspentTxOutFromRef` and `getUtxoutFromRef`

Reply:
Should we rename everything in plutus-contract then?

Reply:
Yep!